lib/monotonic_clock/select/select.ml

```diff
@@ -18,11 +18,16 @@ let () =
     | [| _; "--system"; system; "-o"; output |] ->
         let system =
           match system with
-          | "linux" | "linux_eabihf" | "linux_elf" | "elf" -> `Linux
+          | "linux" | "elf" -> `Linux
+          | "win32" | "win64" | "mingw64" | "mingw" | "cygwin" -> `Windows
```

Comment:
Should we keep `windows` or is this value just a figment of my imagination 😄 ?

Reply:
@dra27 a gently ping, I will try to cut a release as soon as I can.

Reply:
Sorry for missing this - indeed, I'm afraid the constant `windows` was a figment of your imagination 😁